compiler-cli/src/new.rs

```diff
@@ -179,64 +172,79 @@ jobs:
       - uses: actions/checkout@v3
       - uses: erlef/setup-beam@v1
         with:
-          otp-version: "{}"
-          gleam-version: "{}"
-          rebar3-version: "{}"
-          # elixir-version: "{}"
+          otp-version: "{ERLANG_OTP_VERSION}"
+          gleam-version: "{gleam_version}"
+          rebar3-version: "{REBAR3_VERSION}"
+          # elixir-version: "{ELIXIR_VERSION}"
       - run: gleam deps download
       - run: gleam test
       - run: gleam format --check src test
 "#,
-                ERLANG_OTP_VERSION, self.gleam_version, REBAR3_VERSION, ELIXIR_VERSION,
-            ),
-        )
+            )),
+            _ => None,
```

Comment:
Remove this `_` please. We want to match all variants explicitly so that when a new variant is added then the Rust compiler prompts the programmer to update this.